lib/msal-common/src/cache/entities/AccountEntity.ts

```diff
@@ -194,8 +194,20 @@ export class AccountEntity {
             throw ClientAuthError.createInvalidCacheEnvironmentError();
         }
 
+        if (idToken) {
+            // How do you account for MSA CID here?
+            const localAccountId = !StringUtils.isEmpty(idToken.claims.oid)
```

Comment:
btw it could be static method, because now you have two places, where you generate localAccountId